internal/adapters/events_forwarder/noop_forwarder/noop_forwarder_test.go

```diff
@@ -0,0 +1,217 @@
+// MIT License
+//
+// Copyright (c) 2021 TFG Co
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package noop_forwarder
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/topfreegames/maestro/internal/core/entities/forwarder"
+	"github.com/topfreegames/maestro/internal/core/ports/errors"
+	pb "github.com/topfreegames/protos/maestro/grpc/generated"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/topfreegames/maestro/internal/adapters/forwarder_grpc/mock"
+	"github.com/topfreegames/maestro/internal/core/entities/events"
+)
+
+var noopForwarderAdapter *noopForwarder
+var forwarderGrpcMock *mock.MockForwarderGrpc
+var mockCtrl *gomock.Controller
+
+func TestForwardRoomEvent_Arbitrary(t *testing.T) {
```

Comment:
Missing test scenario for when eventAttributes.EventType is not Ping nor Arbitrary